com.unity.render-pipelines.high-definition/Runtime/Lighting/LightCookieManager.cs

```diff
@@ -237,25 +280,126 @@ public Vector4 Fetch2DCookie(CommandBuffer cmd, Texture cookie)
             return scaleBias;
         }
 
-        public Vector4 FetchAreaCookie(CommandBuffer cmd, Texture cookie)
+        public Vector4 FetchAreaCookie(CommandBuffer cmd, Texture cookie, Texture ies, ref Texture emissive)
+        {
+            int width  = (int)Mathf.Max(cookie.width, ies.height);
+            int height = (int)Mathf.Max(cookie.width, ies.height);
+
+            if (width < k_MinCookieSize || height < k_MinCookieSize)
+                return Vector4.zero;
+
+            if (!m_CookieAtlas.IsCached(out var scaleBias, cookie, ies) && !m_NoMoreSpace)
+                Debug.LogError($"Area Light cookie texture {cookie} & {ies} can't be fetched without having reserved. You can try to increase the cookie atlas resolution in the HDRP settings.");
```

Comment:
Same remark as above, poking @JordanL8

Reply:
"Unity cannot fetch the 2D Light cookie textures: {cookie} and {ies} because they are not on the cookie atlas. To resolve this, open your HDRP Asset and increase the resolution of the cookie atlas."